src/Operations.jl

```diff
@@ -2736,6 +2736,25 @@ function print_status(env::EnvCache, old_env::Union{Nothing,EnvCache}, registrie
                 printstyled(io, pkg_str; color=Base.warn_color())
             end
         end
+        # show if loaded version and version in the manifest doesn't match
+        pkg_spec = something(pkg.new, pkg.old)
+        pkgid = Base.PkgId(pkg.uuid, pkg_spec.name)
+        m = get(Base.loaded_modules, pkgid, nothing)
+        if m isa Module && pkg_spec.version !== nothing
+            loaded_path = pathof(m)
+            env_path = Base.locate_package(pkgid) # nothing if not installed
```

Comment:
There are so many ways to get the path to a package, could also for example use `pkgorigins` but this should be fine as well.

Reply:
`pkgorigins` only works for loaded packages? This line's the src as found based on the env, i.e. what loading would find. I thought `locate_package` would be the closest to the actual loading mechanics? If there's a more proper way then let's use it